package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "codesplain",
-  "version": "1.0.0",
+  "version": "0.5.0",
```

Comment:
And this can be overwritten after a release build.  We should set it as 0.6.0 now